src/com/owncloud/android/lib/resources/activities/GetRemoteActivitiesOperation.java

```diff
@@ -68,17 +69,27 @@
 
     private static final String NODE_DATA = "data";
 
+    private String nextUrl = "";
+
+    public void setNextUrl(String url) {
+        nextUrl = url;
+    }
+
     @Override
     protected RemoteOperationResult run(OwnCloudClient client) {
         RemoteOperationResult result = null;
         int status = -1;
         GetMethod get = null;
-        ArrayList<Object> activities;
+        ArrayList<Activity> activities;
         String url;
-        if (client.getOwnCloudVersion().compareTo(OwnCloudVersion.nextcloud_12) >= 0) {
-            url = client.getBaseUri() + OCS_ROUTE_V12_AND_UP;
+        if (nextUrl.isEmpty()) {
+            if (client.getOwnCloudVersion().compareTo(OwnCloudVersion.nextcloud_12) >= 0) {
+                url = client.getBaseUri() + OCS_ROUTE_V12_AND_UP;
+            } else {
+                url = client.getBaseUri() + OCS_ROUTE_PRE_V12;
+            }
         } else {
-            url = client.getBaseUri() + OCS_ROUTE_PRE_V12;
+            url = nextUrl;
         }
         Log_OC.d(TAG, "URL: " + url);
```

Comment:
Remove.

Reply:
I think this makes sense for debugging?

But I will remove all the others

Reply:
👍